### DISCUSSION TITLE
[Feature] Add ability to delete assets from Web UI, and have changes reflected on synced devices

### DISCUSSION THREAD

**Discussion by JosephM101** [immich-app/immich#14212] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

**My request:** I want a feature where the Immich app could detect if an asset that is stored locally on a device; an asset that was _previously uploaded_ to the server, no longer exists on the server. If this is true, show me a dialog or something listing all of the affected assets. The dialog should allow for reviewing what was deleted from the server, and (selectively) either re-upload the assets (maybe it was an accident) or delete them from internal storage.

Here's an example of what the dialog might say:
|             | Description                                                                                                                               |
|-------------|-------------------------------------------------------------------------------------------------------------------------------------------|
| **Title**   | Asset conflict                                                                                                                            |
| **Message** | 4 local assets previously uploaded to the Immich server were deleted from the server by another device. Please review the assets below.   |
|             | *Show a list of affected assets with thumbnails. For each asset, add two icon buttons (a trashcan icon, and a cloud-upload icon)*   |
| **Actions** | *trash-can* - Remove the asset from local device storage<br>*cloud-upload* - Re-upload the asset to the Immich server |

<br>

### Why I'm requesting this feature

I have only one phone that syncs assets with my server. It's a 128GB Pixel 4a (5G). If I opt to "_nuke_" an asset from the Immich app on my phone, they are deleted from both the server and the device's storage. If I delete an asset from the web UI that is still available locally on my phone, the phone will respond by re-uploading the asset the next time the Immich app is opened or performs a sync. In my case, this is unwanted behavior.

I feel like this would make asset management from multiple devices/interfaces much easier. I don't always remove local copies of assets from my phone because my home internet speeds aren't great, and reviewing locally-stored assets from something like the Google Photos app is much faster (in my case) while they're new/still relevant to me.

This is my first feature request. I think it'll be very useful. Hope it is!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#3594
